app/uk/gov/hmrc/economiccrimelevyregistration/views/CheckYourAnswersView.scala.html

```diff
@@ -55,7 +55,7 @@ <h2 class="govuk-heading-m">@messages("checkYourAnswers.organisationDetails")</h
             @govukSummaryList(viewModel.organisationDetails())
         }
 
-        @if(viewModel.hasSecondContact) {
+        @if(viewModel.hasSecondContact && !viewModel.hasSecondContactDetailsPresentChanged) {
```

Comment:
Make this it's own appropriately named viewModel property/val

Reply:
hasSecondContactDetailsPresentChanged is already it's own val in TrackRegistrationChanges and is unit tested